packages/react/src/components/test-utils.ts

```diff
@@ -16,3 +17,7 @@ export const inputPhone = (
   const input = screen.getByPlaceholderText(inputPlaceholder);
   fireEvent.change(input, { target: { value } });
 };
+
+export const sdkNotReadyStates = Object
+  .values(SDKState)
+  .filter((state): state is Exclude<SDKState, SDKState.Ready> => state != SDKState.Ready)
```

Comment:
This is a practical example of where `enum` is helpful, thanks to this we avoided duplicating the states into an array literal that is destined to go stale.

The anti pattern:
```ts
type SDKState = "a" | "b" | "ready"

const sdkNotReadyStates: Exclude<SDKState, "ready"> = [
  "a",
  "b"
]

// looks ok so far, except above is exclusive assertion and not an inclusive one
// it does not assert that all members of SDKState must be present, but they are
// at the time of writing

// add another sdk state
type SDKState = "a" | "b" | "c" | "ready"

// 💥 our tests miss an edge case, and there are no type errors to warn us.
```

Thanks to using an enum, we actually reference the source of truth so we always have an up-to-date list of states.

Reply:
Great idea, this satisfies both arguments!

Reply:
I moved both `sdkStates`, `SDKState` and `sdkNotReadyStates` into `domain/sdk-state.ts`. I think it's a good idea to keep these colocated.